src/Lykke.AlgoStore.CSharp.AlgoTemplate.Models/Repositories/StatisticsRepository.cs

```diff
@@ -13,28 +14,50 @@ namespace Lykke.AlgoStore.CSharp.AlgoTemplate.Models.Repositories
     public class StatisticsRepository : IStatisticsRepository
     {
         private readonly INoSQLTableStorage<StatisticsEntity> _table;
+        private readonly INoSQLTableStorage<StatisticsSummaryEntity> _tableSummary;
 
-        public static readonly string TableName = "Statistics";
+        public static readonly string TableName = "AlgoStatistics";
```

Comment:
Statistics will be about Algo Instance, so maybe we should have AlgoInstanceStatistics? it's just a thought.